offset_manager.go

```diff
@@ -446,13 +450,26 @@ func (om *offsetManager) newBrokerOffsetManager(broker *Broker) *brokerOffsetMan
 func (bom *brokerOffsetManager) mainLoop() {
 	for {
 		select {
+		case <-bom.commitForcedTimer.C:
+			if len(bom.subscriptions) > 0 {
```

Comment:
We should also check `bom.autoCommit` here too, if the commits are manual then we shouldn't interfere